roxctl/generate/netpol/command.go

```diff
@@ -0,0 +1,26 @@
+package netpol
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/netpol/generate"
+)
+
+// Command defines the generate netpol command tree
+func Command(cliEnvironment environment.Environment) *cobra.Command {
+	cmd := generate.NewNetpolGenerateCmd(cliEnvironment)
+	c := &cobra.Command{
+		Use:   "netpol <folder-path>",
+		Short: cmd.ShortText(),
+		Long:  cmd.LongText(),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			cliEnvironment.Logger().WarnfLn("Command 'generate netpol' is deprecated. Use 'netpol generate' instead.")
+			return cobra.ExactArgs(1)(cmd, args)
+		},
+		RunE: func(c *cobra.Command, args []string) error {
+			return errors.Wrap(cmd.RunE(c, args), "running 'generate netpol' command")
```

Comment:
nit: not sure we need the wrap here? or is it because of the linter complaining that we have unwrapped errors? If we need wrapping, let's just call it "generating netpols" or something like this, the exact command isn't really something we require here IMO.

Reply:
Yes, that was a linter suggestion reported as style issue. I will simplify the wrapping string